services/templates/templates.service.ts

```diff
@@ -0,0 +1,118 @@
+import { GlobalCore } from "@/core/module/module.types";
+import { createClient, SupabaseClient } from "@supabase/supabase-js";
+import config from "@/resources/utils/config";
```

Comment:
Can  you create the `templates-mock.service.ts` and add the templates function. to the seeder?

Reply:
Please clarify what you mean by `add templates function to seeder`

Reply:
<img width="715" alt="image" src="https://github.com/user-attachments/assets/45fda364-445d-4065-9619-f03c4d29d23e">
Check the `seed.ts` class :D

Using faker library generate a dynamic template, thats helpfull for testing